website/content/api-docs/index.mdx

```diff
@@ -90,14 +86,21 @@ $ curl \
 
 ## API Operations
 
-With few documented exceptions, all request body data and response data from
-Vault is via JSON. Vault will set the `Content-Type` header appropriately but
-does not require that clients set it.
+All request data, body and response data to and from Vault is JSON except in a 
+few exceptional cases that are documented. Vault sets the `Content-Type` header 
```

Comment:
I would omit the last bit about being `documented` unless  you can provide a link to said documentation.

Reply:
Ok let me see what I can do here - the original and current docs do note the same exception but you are right without linking it's not helpful - all those other cases are by user request in any case (I was thinking like some PKI endpoints that provide binary).